WebApp/src/Interface/IRace.tsx

```diff
@@ -0,0 +1,3 @@
+interface IRace {
```

Comment:
since race will go to both player race and Monster race, maybe also include size and speed (since both share this)